src/ring_theory/jacobson_ideal.lean

```diff
@@ -0,0 +1,105 @@
+/-
+Copyright (c) 2020 Devon Tuma. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau, Devon Tuma
+-/
+import ring_theory.ideal_operations
+
```

Comment:
Please add a [module docstring](https://leanprover-community.github.io/contribute/style.html#module-docstrings) for this new file.